logicrunner/logicrunner_test.go

```diff
@@ -48,7 +48,7 @@ import (
 
 var icc = ""
 var runnerbin = ""
-var parallel = true
+var parallel = false
```

Comment:
We shouldn't accept this change without explanation. What has happened with parallel tests that you had to disable them?

Reply:
thanx, you have found it